server/grpc/gogoreflection/fix_registration.go

```diff
@@ -0,0 +1,143 @@
+package gogoreflection
+
+import (
+	"bytes"
+	"compress/gzip"
+	"fmt"
+	"reflect"
+
+	_ "github.com/gogo/protobuf/gogoproto" // required so it does register the gogoproto file descriptor
+	gogoproto "github.com/gogo/protobuf/proto"
+
+	// nolint: staticcheck
+	"github.com/golang/protobuf/proto"
+	dpb "github.com/golang/protobuf/protoc-gen-go/descriptor"
+	_ "github.com/regen-network/cosmos-proto" // look above
+)
+
+var importsToFix = map[string]string{
+	"gogo.proto":   "gogoproto/gogo.proto",
+	"cosmos.proto": "cosmos_proto/cosmos.proto",
+}
+
+// fixRegistration is required because certain files register themselves in a way
+// but are imported by other files in a different way.
```

Comment:
Could you update this comment please? 
* _themselves in a way_  --> _themselves in one way_
* please provide more details, maybe an example?

Reply:
it's explained down below in the note